src/lib.rs

```diff
@@ -1095,6 +1096,7 @@ pub enum WindowEvent {
     Scroll(f64, f64),
     Key(Key, Scancode, Action, Modifiers),
     Char(char),
+    Dropped(Vec<PathBuf>),
```

Comment:
Just changed it, also fixed the events example which was broken since it wasn't testing for the new event on a match